src/common/mlocal.c

```diff
@@ -103,11 +103,16 @@ char *relfile(char *prog, char *mod) {
 
    canonize(buf);			/* canonize result */
 
+#if __clang__
+   /* clang moans about extra parentheses [-Wparentheses-equality] */
+   if (mod[strlen(mod)-1] == '/')	/* if trailing slash wanted */
+#else                    /* clang */
    if ((mod[strlen(mod)-1] == '/')	/* if trailing slash wanted */
 #if MSDOS
        ||(mod[strlen(mod)-1] == '\\')
 #endif
        )
+#endif                    /* clang */
```

Comment:
Why not just use -Wno-parentheses-equality, which we do anyway. The new if statement suggests that clang and MSDOD (windows?) are mutually exclusive which not the case. We need a better fix fir this one.

Reply:
I just checked Appendix H of the IB and, the comment there is 
        MSDOS    IU        build on Microsoft DOS and descendants (includes Windows, LOL)
I'll suppress the warning with a pragma (and include a reminder to the unwary reader that MSDOS includes WINDOWS).